src/unittest/test_utilities.cpp

```diff
@@ -253,6 +255,23 @@ void TestUtilities::testUTF8()
 		== "the shovel dug a crumbly node!");
 }
 
+void TestUtilities::testRemoveEscapes()
+{
+	UASSERT(remove_enriched_text_escapes<wchar_t>(
```

Comment:
I think it is because the arguments are constant strings, which are interpreted by the compiler as ```const wchar_t *``` and not ```str::wstring```